contracts/TokenHandler.sol

```diff
@@ -4,22 +4,27 @@ import "../installed_contracts/zeppelin-solidity/contracts/ownership/Ownable.sol
 import "../installed_contracts/zeppelin-solidity/contracts/token/ERC20/ERC20Basic.sol";
 
 contract TokenHandler is Ownable {
-
     address public targetToken;
 
-    constructor ( address _targetToken) public Ownable() {
+    constructor (address _targetToken) public Ownable() {
+        require(_targetToken != 0x0, "Must specify a target token!");
```

Comment:
we want to allow the distributor be deployed before or after the token, should the address be needed when deploying the token contract